openid-sse-framework-1_0.txt

```diff
@@ -707,29 +681,22 @@ Tulshibagwale, et al.   Expires December 10, 2021              [Page 12]
    correctly validate MUST be aborted and the information that failed to
    validate MUST NOT be used.
 
-   The "issuer" value returned MUST be identical to the Issuer URL that
+   The issuer value returned MUST be identical to the Issuer URL that
    was directly used to retrieve the configuration information.  This
-   MUST also be identical to the "iss" Claim value in Security Event
+   MUST also be identical to the iss Claim value in Security Event
    Tokens issued from this Transmitter.
 
 7.  Management API for SET Event Streams
 
    This section defines an HTTP API to be implemented by Event
-   Transmitters and that can be used by Event Receivers to query and
-   update the Event Stream configuration and status, to add and remove
-   subjects and to trigger verification.
+   Transmitters which can be used by Event Receivers to create and
+   delete one or more Event Streams, query and update the configuration
```

Comment:
Since we are talking about multiple event streams now, we should probably write a new section that defines what an event stream is.

Reply:
Added